.github/workflows/documentation.yml

```diff
@@ -0,0 +1,28 @@
+name: Documentation
+on:
+  release:
+    types:
+      - published
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Generate Documentation
+        uses: SwiftDocOrg/swift-doc@master
+        with:
+          base-url: /swift-composable-architecture/
+          format: html
+          inputs: Sources/ComposableArchitecture
+          module-name: ComposableArchitecture
+          output: Documentation
+      - name: Update Permissions
+        run: 'sudo chown --recursive $USER Documentation'
```

Comment:
Can remove this whenever https://github.com/SwiftDocOrg/swift-doc/pull/146 is merged.

Reply:
@stephencelis Done!